--- a/PKGBUILD	2018-08-16 18:04:59.000000000 +0700
+++ b/PKGBUILD	2018-08-17 12:05:37.263285515 +0700
@@ -3,7 +3,7 @@
 # Maintainer: Tobias Powalowski <tpowa@archlinux.org>
 # Maintainer: Thomas Baechler <thomas@archlinux.org>
 
-pkgbase=linux               # Build stock -ARCH kernel
+pkgbase=linux-apparmor               # Build stock -ARCH kernel
 #pkgbase=linux-custom       # Build kernel with a different name
 _srcver=4.18.1-arch1
 pkgver=${_srcver//-/.}
@@ -54,6 +54,22 @@
 
   msg2 "Setting config..."
   cp ../config .config
+
+  cat <<EOF >> .config
+CONFIG_SECURITY_APPARMOR=y
+CONFIG_SECURITY_APPARMOR_BOOTPARAM_VALUE=1
+CONFIG_SECURITY_APPARMOR_HASH=y
+CONFIG_SECURITY_APPARMOR_HASH_DEFAULT=y
+CONFIG_SECURITY_APPARMOR_COMPAT_24=y
+CONFIG_DEFAULT_SECURITY_APPARMOR=y
+CONFIG_DEFAULT_SECURITY="apparmor"
+
+# CONFIG_NETFILTER_XT_TARGET_AUDIT is not set
+# CONFIG_SECURITY_SELINUX is not set
+# CONFIG_DEFAULT_SECURITY_DAC is not set
+# CONFIG_SECURITY_APPARMOR_DEBUG is not set
+EOF
+
   make olddefconfig
 
   make -s kernelrelease > ../version
